Formula/orientdb.rb

```diff
@@ -56,15 +61,16 @@ def caveats
 
   test do
     ENV["CONFIG_FILE"] = "#{testpath}/orientdb-server-config.xml"
+    ENV["ORIENTDB_ROOT_PASSWORD"] = "changeme"
 
     cp "#{libexec}/config/orientdb-server-config.xml", testpath
     inreplace "#{testpath}/orientdb-server-config.xml", "</properties>", "  <entry name=\"server.database.path\" value=\"#{testpath}\" />\n    </properties>"
 
     system "#{bin}/orientdb", "start"
-    sleep 4
+    sleep 3
 
     begin
-      assert_match "OrientDB Server v.2.2.5", shell_output("curl -I localhost:2480")
+      assert_match "3 item(s) found.", shell_output("#{bin}/orientdb-console \" CREATE DATABASE remote:localhost/test-brew root changeme  plocal; disconnect; connect remote:localhost/test-brew admin admin; select from OUser \"")
```

Comment:
Could probably have a simpler call here; just want to verify it hasn't crashed. Ideally could you split this onto multiple lines with `EOS.undent` and/or slim it down a bit?